ts/annotations/annotations.src.ts

```diff
@@ -1057,12 +1088,17 @@ merge(
          * @private
          */
         init: function (this: Highcharts.Annotation): void {
+            const chart = this.chart,
+                defer = animObject(this.options.animation).defer;
+
             this.linkPoints();
             this.addControlPoints();
             this.addShapes();
             this.addLabels();
             this.addClipPaths();
             this.setLabelCollector();
+            this.deferTime = defined(defer) ? defer : getDeferTime(chart);
```

Comment:
`defer` should now be always defined, right? Then:
- `defer = animObject(this.options.animation).defer;` -> `defer = 0`
- `this.deferTime = defined(defer) ? defer : getDeferTime(chart);` -> `this.deferTime = 0`